File: problemas/iniciante/2344/2344.js
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input = "";

process.stdin.on('data', (chunk) => {
    input += chunk;
});

process.stdin.on('end', () => {
    const N = parseInt(input.trim().split('\n'));

    if (N > 85) {
        console.log("A");
    }
    else if (N > 60) {
        console.log("B");
    }
    else if (N > 35) {
        console.log("C");
    }
    else if (N > 0) {
        console.log("D")
    } else {
        console.log("E");
    }
});

File: problemas/iniciante/2344/2344.py
def main():
    N = int(input())

    if (N == 0):
        print("E")
    elif (1 <= N <= 35):
        print("D")
    elif (36 <= N <= 60):
        print("C")
    elif (61 <= N <= 85):
        print("B")
    else:
        print("A")

if __name__ == "__main__":
    main()
